### DISCUSSION TITLE
Win32 path separator on  AsyncFileStreamer::streamFile

### DISCUSSION THREAD

**Discussion by mbsteixeira** [uNetworking/uWebSockets.js#225] [Q&A] [answered]
In the method "AsyncFileStreamer::streamFile",  the   "auto it = asyncFileReaders.find(url);" fails because of the **windows** plataform path separator.
It's necessary a   change from '/' to '\\' , so the search succeeds.

**Comment by ghost** [comment]
Yes one could work on that however it is just a simple example and could be solved in a cross platform way if wanted

**Comment by mbsteixeira** [comment]
Ok.

**Comment by ghost** [comment, marked as answer]
Feel free to PR though
